x-pack/plugins/endpoint/public/applications/endpoint/store/index.ts

```diff
@@ -48,37 +49,37 @@ export const substateMiddlewareFactory = <Substate>(
   };
 };
 
-export const appStoreFactory: (
+export const appStoreFactory: (middlewareDeps?: {
```

Comment:
Could you add a doc comment here explaining the behavior when no middleware deps are passed

Reply:
done